dbfit-java/informix/src/test/java/dbfit/environment/InformixBigDecimalTransformerTest.java

```diff
@@ -0,0 +1,45 @@
+package dbfit.environment;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+import java.math.BigDecimal;
+import java.sql.SQLException;
+
+import dbfit.util.NormalisedBigDecimal;
+
+public class InformixBigDecimalTransformerTest {
+
+    @Test
+    public void transformsTest() {
+        InformixBigDecimalTransformer bdt = new InformixBigDecimalTransformer();
+        BigDecimal inBigDec = new BigDecimal("123.45");
+        NormalisedBigDecimal nbd = new NormalisedBigDecimal(inBigDec);
+        BigDecimal outBigDec = null;
+        try {
+            outBigDec = (BigDecimal) bdt.transform(nbd);
+        } catch (SQLException e) {
+            fail("InformixBigDecimalTransformer threw SQLException transforming NormalisedBigDecimal input");
+        }
+        assertEquals(outBigDec, inBigDec);
+    }
+
+    @Test
+    public void rejectsInputTest() {
+        InformixBigDecimalTransformer bdt = new InformixBigDecimalTransformer();
+        BigDecimal inBigDec = new BigDecimal("123.45");
+        boolean caught = false;
+        String expectedMsg = "InformixBigDecimalTransformer cannot transform objects of type java.math.BigDecimal";
```

Comment:
> Looking at that message now makes me think: we have `InformixBigDecimalTransformer` but it cannot transform `BigDecimal`... Which is maybe a bit weird.

So perhaps `InformixNormalisedBigDecimalTransformer` is a better name. Hmmmm. Then again may be even `NormalisedBigDecimalTransformer` and make it DB-type agnostic. Any view?